### PR TITLE
feat(acot-preset-wcag): add link-has-name

### PR DESCRIPTION
**Pull request by masuP9**
## What does this change?

Add link-has-name rule to acot-preset-wcag

## Screenshots

none.

## What can I check for bug fixes?

none.

## References

- [WCAG 2.1 - 2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/WCAG21/#link-purpose-in-context)
- [WCAG 2.1 - 4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value)

### CODE REVIEW

File: packages/acot-preset-wcag/docs/rules/link-has-name.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# img-has-name
```
Comment (wadackel) [inline packages/acot-preset-wcag/docs/rules/link-has-name.md]:
```suggestion
# link-has-name
```

File: packages/acot-preset-wcag/src/rules/link-has-name.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        const ax = await (window as any).getComputedAccessibleNode(el);
+        return (ax?.name ?? '').trim();
+      });
+
+      context.debug('name: %s', name);
+
+      if (!name) {
+        await context.report({
+          node,
+          message: 'link MUST has name.',
```
Comment (wadackel) [inline packages/acot-preset-wcag/src/rules/link-has-name.ts]:
```suggestion
          message: 'Link MUST has name.',
```

**Review by wadackel — CHANGES_REQUESTED** [review]
Thank you for creating rule!

---

Please execute the following command:

```bash
$ yarn workspace @acot/acot-preset-wcag docgen
```

**Comment by masuP9** [comment]
Sorry, I made PR by wrong branch. I will make another one.

**Pull request closed without merging**